cassava-trait-ontology_with-cropname.obo

```diff
@@ -1820,67 +1819,67 @@ is_a: CO_334:0000037 ! Cassava bacterial blight incidence
 
 [Term]
 id: CO_334:0000181
-name: cassava anthractnose disease incidence in 6-month variable
+name: cassava anthracnose disease incidence in 6-month variable
 def: "Proportion of plants within a plot showing symptoms of cassava anthracnose disease evaluated between four and six months after planting (6MAP)." [CO:curators]
 synonym: "cad6i" EXACT []
 synonym: "CADisInc_Comp_r_T6m" EXACT []
 relationship: variable_of CO_334:0000038 ! Cassava anthracnose disease incidence
-relationship: variable_of CO_334:0010258 ! Counting:cassava anthractnose disease incidence in 6-months_method
+relationship: variable_of CO_334:0010258 ! Counting:cassava anthracnose disease incidence in 6-months_method
 relationship: variable_of CO_334:0100582 ! ratio
 is_a: CO_334:0000038 ! Cassava anthracnose disease incidence
 
 [Term]
 id: CO_334:0000182
-name: cassava anthractnose disease incidence in 9-month variable
+name: cassava anthracnose disease incidence in 9-month variable
 def: "Proportion of plants within a plot showing symptoms of cassava anthracnose disease evaluated between seven and nine months after planting (9MAP)." [CO:curators]
 synonym: "cad9i" EXACT []
 synonym: "CADisInc_Comp_r_T9m" EXACT []
 relationship: variable_of CO_334:0000038 ! Cassava anthracnose disease incidence
-relationship: variable_of CO_334:0010259 ! Counting:cassava anthractnose disease incidence in 9-months_method
+relationship: variable_of CO_334:0010259 ! Counting:cassava anthracnose disease incidence in 9-months_method
 relationship: variable_of CO_334:0100582 ! ratio
 is_a: CO_334:0000038 ! Cassava anthracnose disease incidence
 
 [Term]
 id: CO_334:0000183
-name: cassava anthractnose disease incidence in12-month variable
+name: cassava anthracnose disease incidence in12-month variable
 def: "Proportion of plants within a plot showing symptoms of cassava anthracnose disease evaluated between ten and twelve months after planting (12MAP)." [CO:curators]
 synonym: "cad12i" EXACT []
 synonym: "CADisInc_Comp_r_T12m" EXACT []
 relationship: variable_of CO_334:0000038 ! Cassava anthracnose disease incidence
-relationship: variable_of CO_334:0010260 ! Counting:cassava anthractnose disease incidence in 12-months_method
+relationship: variable_of CO_334:0010260 ! Counting:cassava anthracnose disease incidence in 12-months_method
 relationship: variable_of CO_334:0100582 ! ratio
 is_a: CO_334:0000038 ! Cassava anthracnose disease incidence
 
 [Term]
 id: CO_334:0000184
-name: cassava anthractnose disease severity in 6-month variable
+name: cassava anthracnose disease severity in 6-month variable
```

Comment:
need to apply this change to the main obo file too

Reply:
Yes, already discussed with @marieALaporte